examples/nodejs/aws/eventbridge/run-typescript-cli-app.sh

```diff
@@ -34,14 +32,19 @@ if [ -z "$AWS_ACCESS_KEY_ID" ] || [ -z "$AWS_SECRET_ACCESS_KEY" ] || [ -z "$AWS_
     exit 1
 fi
 
+set -e
+set -x
```

Comment:
@rishtigupta I moved these around a bit to try to avoid logging creds to the shell